index.html

```diff
@@ -217,6 +217,21 @@ <h4>CSS</h4>
             </li>
             <img src="./images/Separator1.png" alt="" class="separator2">
         </ul>  
+    </section>
+                                        <!-- page 6 -->
+                                    <!-- Section 1 : Contact form -->
+    <section class="contact">
+        <img src="/images/A.png" alt="" class="contact-img">
+        <img src="/images/contact form shapes-mobile@2x.png" alt="" class="contact-img2">
```

Comment:
- [ ] Your background images are not visible to all browsers as you have missed the relative paths in the src attributes. Also for the best practices, please use CSS background-images property to use them rather than using them in HTML.
![image](https://user-images.githubusercontent.com/91486791/220361772-5025a5b7-c89b-4365-8d74-853249c92e4e.png)